website/src/components/product/CaseStudiesList.tsx

```diff
@@ -85,30 +83,42 @@ const CASESTUDIES: {
     {
         name: 'Thorn',
         logo: '/external-logos/thorn-logo.svg',
-        title: 'Thorn deprecates legacy code safely',
+        title: 'Thorn deprecates legacy code safely.',
         url: '/case-studies/we-are-thorn',
     },
+    {
+        name: 'Nutanix',
+        logo: '/external-logos/nutanix-logo.svg',
+        title: 'Nutanix fixed Log4j quickly and confidently with Sourcegraph.',
+        url: '/case-studies/nutanix-fixed-log4j-with-sourcegraph',
+    },
 ]
 
-export const CaseStudiesSection: React.FunctionComponent<{ className?: string }> = ({ className = '' }) => (
+export const CaseStudiesList: React.FunctionComponent<{ className?: string; listLength?: number }> = ({
+    className = '',
+    listLength = CASESTUDIES.length,
+}) => (
     <div id="customers" className={`case-studies-section container-fluid ${className}`}>
         <div className="card-deck">
-            {CASESTUDIES.map((study, i) => (
-                <div key={i} className="col-lg-4 mb-6">
+            {CASESTUDIES.slice(0, listLength).map((study, i) => (
+                // 3 Cards per row if this is the Case Study index pg, otherwise it's being re-used as a preview w/ 2 Cards per row
+                <div key={i} className={`col-lg-${listLength === CASESTUDIES.length ? 4 : 6} mb-6`}>
```

Comment:
Sure! I missed that part. I can change the list component to cards so it can be more easily styled by it's parent. That'll address your other comment below about the card borders.